src/modules/LocalStorage_UI.js

```diff
@@ -0,0 +1,115 @@
+      localStorage.setItem('todolists', JSON.stringify(todolists));
+    } else {
+      todolists = JSON.parse(localStorage.getItem('todolists'));
+    }
+    return todolists;
+  }
+
+  static addToDoLists(item) {
+    const todolists = LocalStorage.getToDoLists();
+    todolists.push(item);
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+
+  static removeToDoLists(newdesc) {
+    const todolists = LocalStorage.getToDoLists();
+
+    todolists.forEach((item, index) => {
+      if (item.desc === newdesc) {
+        todolists.splice(index, 1);
+      }
+    });
+
+    // RESET INDEX
+    LocalStorage.resetIndex();
+
+    // UPDATE DESCRIPTION
+    LocalStorage.updateDesc();
+
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+
+  static resetIndex() {
+    todolists.forEach((item, index) => {
+      item.index = index + 1;
+    });
+  }
+
+  static updateDesc(desc, index) {
+    for (let i = 0; i < todolists.length; i += 1) {
+      if (todolists[i].index === +index) {
+        todolists[i].desc = desc;
+        localStorage.setItem('todolists', JSON.stringify(todolists));
+      }
+    }
+  }
+}
+
+class UserInterface {
+  static showToDoLists() {
+    const todolists = LocalStorage.getToDoLists();
+
+    todolists.forEach((item) => UserInterface.addToDoLists(item));
+
+    // REMOVE TO DO LIST
+    const dustbin = document
+      .querySelector('#list')
+      .querySelectorAll('.fa-trash-can');
+
+    dustbin.forEach((bin) => {
+      bin.addEventListener('click', (e) => {
+        // Remove item from UserInterface
+        UserInterface.removeToDoLists(e.target.parentElement);
+        // Remove item from LocalStorage
+        LocalStorage.removeToDoLists(
+          e.target.parentElement.previousElementSibling.children[1].value,
+        );
+      });
+    });
+
+    // UPDATE DESCRIPTION
+    const desc = document.querySelector('#list').querySelectorAll('.desc');
+
+    desc.forEach((elt) => {
+      elt.addEventListener('change', (e) => {
+        LocalStorage.updateDesc(
+          e.target.value,
+          e.target.parentElement.parentElement.id,
+        );
+      });
+    });
+  }
+
+  static addToDoLists(item) {
+    const listContent = document.querySelector('#list');
+    const listElt = document.createElement('li');
+    listElt.classList = 'new';
+    listElt.id = `${item.index}`;
+    listElt.innerHTML += `
+      <span class="inputs"><input class="checkbox" type="checkbox" /><input class="desc" type="text" value="${item.desc}" /></span>
+      <button class="text-btn" type="button">
+        <i class="fa-solid fa-trash-can"></i>
+      </button>
+      `;
+    listContent.appendChild(listElt);
+  }
+
+  static removeToDoLists(elt) {
+    if (elt.classList.contains('text-btn')) {
+      elt.parentElement.remove();
+    }
+    localStorage.setItem('todolists', JSON.stringify(todolists));
+  }
+
+  static clearFields() {
+    document.querySelector('.text').value = '';
+  }
+}
```

Comment:
- [Optional] Good job making the changes @miltonHenschel, But as you can see from my screenshot I cannot delete tasks unless reload the page. Also, the completed (checkbox) state is not saved. This will make your app functionality incomplete. Please consider adding this functionality.

https://user-images.githubusercontent.com/69077061/213254942-365e3deb-2927-4f33-8d8d-69537f703b03.mp4